crates/torii/libp2p/src/server/mod.rs

```diff
@@ -38,6 +38,8 @@ use crate::server::events::ServerEvent;
 use crate::typed_data::PrimitiveType;
 use crate::types::Message;
 
+pub(crate) const LOG_TARGET: &str = "torii::relay::server";
```

Comment:
@glihm Notice that this file has multiple log target. I think ideally it should have only one right? That's why I replace them all with just this one. Let me know if that's ok or not. Thanks